doc_swift_ui_gallery/swift_ui_gallery/swift_ui_gallery/Localizable.xcstrings

```diff
@@ -53,6 +53,9 @@
     },
     "All styles" : {
 
+    },
+    "AngularGradient" : {
```

Comment:
Do you know why content in this file changed?

Reply:
Xcode doing its magic. Since I pulled in the latest changes, there were new localizable strings. They are automatically added to the default localization file at build time